### PR TITLE
first version of the footer component

### PR DESCRIPTION
**Pull request by roohibengani**
This PR contains the initial version of the footer component.
Things left to do:

- Update the footer social icons as per the figma
- update the logo image and footer image as per the images provided.
- Contact Oly button will be updated post changes in the typography files

Test URLs:
- Before: https://main--mmsg-oly--aemsites.hlx.live/
- After: https://footer--mmsg-oly--aemsites.hlx.live/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,243 @@
 footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+  background-color: var(--bg-color-paper);
+  font-size: var(--font-size-xs);
+  color: var(--color-black);
+  line-height: 1.5
```
Comment (mojojain) [inline blocks/footer/footer.css:5]: Add a semi-colon here.

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,14 +1,243 @@
...
+        height: 1.5rem;
+      }
+    }
+  }
+
+  .footer-nav-div {
+    >p {
+      &:nth-child(2) {
+        font-weight: 400;
+        line-height: 1rem;
```
Comment (mojojain) [inline blocks/footer/footer.css:95]: Line-height should be 1.5 rem.

File: blocks/footer/footer.css (hunk 1 above)
Comment (roohibengani) [reply blocks/footer/footer.css]: Done

File: blocks/footer/footer.css (hunk 2 above)
Comment (roohibengani) [reply blocks/footer/footer.css]: Done

**Review by mrugesh-adobe — APPROVED** [review]

**Pull request merged**